tests/frontier/opcodes/test_swap.py

```diff
@@ -0,0 +1,130 @@
+"""
+A State test for the set of `SWAP*` opcodes.
+Ported from: https://github.com/ethereum/tests/blob/develop/src/GeneralStateTestsFiller/VMTests/vmTests/swapFiller.yml.
+"""
+
+import pytest
+
+from ethereum_test_forks import Fork, Frontier, Homestead
+from ethereum_test_tools import Account, Alloc, Environment
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools import StateTestFiller, Storage, Transaction, Bytecode
```

Comment:
```suggestion
from ethereum_test_tools import StateTestFiller, Transaction, Bytecode
```
`Storage` is unused.